lib/super_digit.rb

```diff
@@ -1,15 +1,17 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(log n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
This is O( (log n)^2) because the summation is O(log n), and it's inside a recursive method which repeats log n times.